File: sample-projects/WebApiNet/WebApiNet/Models/Testing.cs
namespace WebApiNet.Models;

using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

[Table("testing")]
public class Testing  
{
	[Key]
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)]
	[Column("id")]
	public long Id { get; set; }

	[Column("name")]
	public string Name { get; set; }

	[Column("email")]
	public string Email { get; set; }

	[Column("phone")]
	public string Phone { get; set; }
	

}

File: sample-projects/WebApiSpring/src/main/java/com/labs/webapispring/models/Testing.java
package com.labs.webapispring.models;

import jakarta.persistence.*;

@Entity
@Table(name="testing")
public class Testing  {
	@Id
	@GeneratedValue(strategy=GenerationType.IDENTITY)
	@Column(name="id")
	private Long id;

	@Column(name="name")
	private String name;

	@Column(name="email")
	private String email;

	@Column(name="phone")
	private String phone;
	

	public Long getId() {
	   return id;
	}

	public void setId(Long id) {
	   this.id = id;
	}
	
	public String getName() {
	   return name;
	}

	public void setName(String name) {
	   this.name = name;
	}
	
	public String getEmail() {
	   return email;
	}

	public void setEmail(String email) {
	   this.email = email;
	}
	
	public String getPhone() {
	   return phone;
	}

	public void setPhone(String phone) {
	   this.phone = phone;
	}

}
